source/puppet/4.1/reference/resources_package_windows.markdown

```diff
@@ -102,6 +102,8 @@ It's a good idea to use the hash notation for file path arguments since they may
 install_options => [ { 'INSTALLDIR' => "${packagedir}" } ]
 {% endhighlight %}
 
+I'm not sure this ^ is necessary as puppet should quote arguments with spaces automatically.
+
```

Comment:
@joshcooper I think the reason to put quotes there is that Puppet will **not** automatically quote the path **unless** it has spaces. Does that sound right? 

But if it doesn't have spaces, it probably doesn't need to be quoted anyways. This could be a case of :lipstick: